recipe/meta.yaml

```diff
@@ -21,17 +21,20 @@ requirements:
     - python
     - pip
     - cython >=0.28.5
-    - numpy
+    - numpy >=1.13.3
+    - wheel
+    - setuptools
   run:
     - python
     - {{ pin_compatible('numpy') }}
-    - scipy
+    - scipy >=0.19.1
     - joblib
     - scikit-learn >=0.23.0
```

Comment:
Their docs says >= 0.14 but their `setup.py` says >= 0.23... Not sure which one we should use.

Reply:
Yeahh.. I got a bit confused by this as well. Let me take another look into it.